### DISCUSSION TITLE
Tanyakan apapun terkait projek terbuka Cuy Universe

### DISCUSSION THREAD

**Discussion by deaafrizal** [deaafrizal/cuyuniverse-forum#70] [Q&A]
### Q&A Discussion

**disini tempat untuk yang ingin bertanya dan HANYA terkaitt projek ini saja.**

**Comment by fadhilin** [comment]
bang ko code css gua ga konek ke html ya hanya beberapa yang konek, pas mau ganti background color ga keubah itu kenapa bang?

**Reply by deaafrizal** [reply]
boleh di sertain screeshot nya disini bro

**Comment by kongis23** [comment]
![kongis](https://github.com/deaaprizal/cuyuniverse-forum/assets/149935703/3a81b9f6-f5f9-4d3d-9ea5-ec6867edfafb)
bang,ko js gua error mulu ya kaya gini

**Comment by nzrl-af** [comment]
Coba ss codenya

Pada Sen, 6 Nov 2023 17.15, kongis23 ***@***.***> menulis:

**Reply by kongis23** [reply]
![kongis 2 0](https://github.com/deaaprizal/cuyuniverse-forum/assets/149935703/1f2d7621-38e4-441c-8096-4d1b69872806)
ini bang

**Reply by kongis23** [reply]
bisa tolong di bantu abang abangku?

**Comment by nzrl-af** [comment]
nama file JS nya coba mungkin salah


Pada Rab, 8 Nov 2023 19.03, kongis23 ***@***.***> menulis:

**Reply by kongis23** [reply]
astaga gua ternyata belum masukin file js nya bang,yhahha kacaw

**Reply by kongis23** [reply]
uda bisa bang tengkyu ya,gua inget karna lu ngmng "file"ni wkwkw

**Comment by wegeida12** [comment]
bang kalau pemula cara belajarnya kaya bagimana
